src/components/Loading/loading.spec.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import Loading from "./index";
+
+describe("Loading Component", () => {
+  test("renders loading spinner and text", () => {
+    const text = "Loading...";
+    render(<Loading text={text} />);
+    const spinner = screen.getByTestId("spinner");
+    const loadingText = screen.getByText(text);
+    expect(spinner).toBeInTheDocument();
+    expect(loadingText).toBeInTheDocument();
```

Comment:
```suggestion
  test("renders loading spinner and text", () => {
    const text = "Loading...";

    render(<Loading text={text} />);
    
    const spinner = screen.getByTestId("spinner");
    const loadingText = screen.getByText(text);
    
    expect(spinner).toBeInTheDocument();
    expect(loadingText).toBeInTheDocument();
```
관심사 별로 개행해서 묶어주는 것이 어떨까 싶습니다😄

Reply:
beforeEach로 Loading 컴포넌트를 렌더하시고 test("renders loading spinner")과 test("renders loading text") 분리하실 수도 있으실 것 같아요.